supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -874,6 +885,16 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 
 *See also*:
 
+[[rolling-stream]]
+==== image:images/yes.png[yes] Rolling stream (noun)
+*Description*: _Rolling Application Streams_ include Red Hat Enterprise Linux tools and applications that are updated frequently. Later versions of these streams replace earlier versions. Therefore, only one version at a time is supported.
```

Comment:
term is "Rolling stream," but description says "Rolling [App] Streams"

- additional word ("[App]")
- cap S versus lc s in term

clarify?

Reply:
Confirmed with SMEs, adding capitalization